apps/spruce/src/analytics/patch/usePatchAnalytics.ts

```diff
@@ -27,8 +28,8 @@ export const usePatchAnalytics = (id: string) => {
   });
   const { status } = eventData?.patch || {};
 
-  return useAnalyticsRoot<Action>("Patch", {
-    patchStatus: status,
-    patchId: id,
+  return useAnalyticsRoot<Action, AnalyticsIdentifier>("Patch", {
+    "patch.status": status || "",
+    "patch.id": id,
```

Comment:
Is this (and other object-related updates) Honeycomb related? Or just better organization?